modules/admin_manual/pages/configuration/server/config_sample_php_parameters.adoc

```diff
@@ -2286,17 +2320,18 @@ Set this property to true if you want to enable debug logging for SMB access.
 'dav.enable.async' => false,
 ....
 
-=== Allow propfind depth infinity
+=== Enable propfind depth infinity requests
+Tell the clients whether `depth=infinity` is allowed for propfind requests.
 
-With this setting that defaults to true, propfind requests will now be streamed to reduce memory usage
-with large responses. It tells the clients whether `depth=infinity` is allowed for propfind requests.
+Streamed infinite depth propfind requests can reduce memory usage
+with large responses.
 For details see: https://datatracker.ietf.org/doc/html/rfc4918#section-10.2
 
 ==== Code Sample
 
 [source,php]
 ....
-'dav.propfind.depth_infinity' => true,
+'dav.propfind.depth_infinity' => false,
```

Comment:
This change to the default value will be in the next release, which is planned to be numbered 10.10.0 - so no backport at all.

Reply:
thanks for the reminder